cgmes/cgmes-model-alternatives/src/test/resources/sample_not_all_tap_changers_have_control/MicroGridTestConfiguration_BC_BE_SSH_V2.xml

```diff
@@ -200,31 +200,31 @@
   <cim:EquivalentInjection rdf:about="#_f7f61a91-eca2-4492-8bd7-9ec2b28fc837">
     <cim:EquivalentInjection.p>-26.805006</cim:EquivalentInjection.p>
     <cim:EquivalentInjection.q>1.489867</cim:EquivalentInjection.q>
-    <cim:EquivalentInjection.regulationStatus>false</cim:EquivalentInjection.regulationStatus>
+    <cim:EquivalentInjection.regulationStatus>OFF</cim:EquivalentInjection.regulationStatus>
```

Comment:
I think this is a mistake but CGMES files should not be modified

Reply:
All cgmes files are reverted